Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "29"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 350,
+          "y": 330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 9295e1a2-7179-4325-8c78-27414e33a3c9
+    type: condition
+    task:
+      id: 9295e1a2-7179-4325-8c78-27414e33a3c9
+      version: -1
+      name: Is the process blocked?
+      description: Check if the CrowdStrike Falcon agent blocked the suspicious process.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "17"
+      Automated Remediation:
+      - "77"
+      'Manual Remediation  ':
+      - "78"
+    separatecontext: false
+    conditions:
+    - label: 'Manual Remediation  '
+      condition:
+      - - operator: in
+          left:
+            value:
+              simple: CrowdStrike.Detection.policyactions.process_blocked
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+        - operator: in
+          left:
+            value:
+              simple: CrowdStrike.Detection.policyactions.operation_blocked
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Endpoint
+                filters:
+                - - operator: in
+                    left:
+                      value:
+                        simple: Endpoint.ID
+                      iscontext: true
+                    right:
+                      value:
+                        simple: incident.agentsid
+                      iscontext: true
+                    ignorecase: true
+                accessor: Status
+                transformers:
+                - operator: uniq
+            iscontext: true
+          right:
+            value:
+              simple: Online
+          ignorecase: true
+      - - operator: greaterThanOrEqual
+          left:
+            value:
+              complex:
+                root: CommandLineAnalysis
+                filters:
+                - - operator: in
+                    left:
+                      value:
+                        simple: CommandLineAnalysis.original_command
+                      iscontext: true
+                    right:
+                      value:
+                        simple: CrowdStrike.Detection.parentprocess.cmdline
+                      iscontext: true
+                accessor: score
+            iscontext: true
+          right:
+            value:
+              simple: "26"
+    - label: Automated Remediation
```

Comment:
for the automated remediation it shouldnt be one of the actions blocked? and not both of them?

Reply:
The automated path will be taken once the process wasn't blocked (means both of them false) + the agent is online + the CMD is malicious